src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -1068,8 +1068,8 @@ static MemorySegment ofAddress(long address) {
      * restricted methods, and use safe and supported functionalities, where possible.
      * <p>
      * On 32-bit platforms, the given address value will be normalized such that the
-     * returned memory segment's method {@link MemorySegment#address() address} will always return
-     * zeros for the upper 32 bits of the address.
+     * upper 32 bits of the {@link MemorySegment#address() address} of returned memory segment
```

Comment:
Ah - late to the party. Is "upper bits" correct here? E.g. doesn't it depend on endianness?

Reply:
I'm partly confusing myself, but we should still draw from javadoc in other areas:
https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/lang/Long.html#highestOneBit(long)
This seems to use highest-order/leftmost for "uppermost bit" :-)